src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +23,29 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+
+    private static final int SIZE = 5;
+
+    public String getUniqueCharacters(String fileName) throws IOException {
+
+        Set<Character> chars = new TreeSet<>();
+        try (BufferedReader br = new BufferedReader(new FileReader(fileName))) {
+            String line;
+            while ((line = br.readLine()) != null) {
```

Comment:
there is also method `read()` that reads the next character and returns its codepoint

Reply:
Should I replace them with each other?